### ISSUE TITLE
std@v1.0.0-rc1 deps missing

### ISSUE THREAD

**Issue by dbohdan** [oakserver/oak#64]
Per https://github.com/denoland/deno/issues/5176, the `v1.0.0-rc1` tag on `std` was a mistake ~and one should use `v0.42.0` (with the same Deno version)~.  `v0.50.0` is now up.

**Comment by sorxrob** [comment]
Hi, I tried this but I'm getting 

`error: Uncaught Error: Import 'https://deno.land/x/oak@v0.50.0/mod.ts' failed: 404 Not Found`

Here's my code

```javascript
import { Application } from "https://deno.land/x/oak@v0.50.0/mod.ts";

const app = new Application();

app.use((ctx) => {
  ctx.response.body = "Hello World!";
});

await app.listen({ port: 8000 });
```

or am I doing it wrong?

**Comment by kitsonk** [comment]
Fixed via 80994ab053989e794eebd49ef812e684f0a517dc
